Backend/BikesRentalServer.WebApi/Dtos/Responses/GetMalfunctionResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using BikesRentalServer.Models;
+
+namespace BikesRentalServer.WebApi.Dtos.Responses
+{
+    public class GetMalfunctionResponse
```

Comment:
This DTO does not meet with RAML schema

![image](https://user-images.githubusercontent.com/24509819/120222383-25a1fa80-c240-11eb-932b-d31c63aca9ca.png)